### PR TITLE
Improve mailing configuratibility

### PR DESCRIPTION
**Pull request by PetrDlouhy**
* Enable HTML e-mails
* Make possible to configure certain e-mail types not to be send

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,11 @@
 django-plans changelog
 ======================
 
+0.9.1
+------
```
Comment (Alir3z4) [inline CHANGELOG:5]: We need to keep the changelog under `0.9.0` since even that `0.9.0` is not release yet.

File: docs/source/settings.rst [hunk 2]
```diff
@@ -236,6 +236,34 @@ Default: ``True``
 
 Boolean value for enabling (default) or disabling the sending of plan related emails.
 
+``PLANS_SEND_EMAILS_NO_SEND_TYPES``
```
Comment (Alir3z4) [inline docs/source/settings.rst:239]: Maybe a another name like `PLANS_SEND_EMAILS_DISABLED_TYPES` ?

**Comment by PetrDlouhy** [comment]
@Alir3z4 I have fixed the conf vars names, the changelog.

**Comment by Alir3z4** [comment]
This looks good to me!
Let's merge it.

Thanks for the great work.

**Pull request merged**